### PR TITLE
Rebuild fix

### PR DESCRIPTION
**Pull request by 0xf8f8ff**
Fixed a bug where a wrong repository could sometimes be fetched when rebuilding a student submission.

### CODE REVIEW

File: web/autograder_service.go [hunk 1]
```diff
@@ -489,13 +489,10 @@ func (s *AutograderService) ApproveSubmission(ctx context.Context, in *pb.Approv
 
 // RefreshSubmission rebuilds the latest student submission for the logged in user
```
Comment (meling) [inline web/autograder_service.go]: Maybe the doc comment needs to be updated to say rebuild for the user specified in ...?

File: public/src/pages/views/LabResultView.tsx [hunk 2]
```diff
@@ -54,8 +55,8 @@ export class LabResultView extends React.Component<ILabInfoProps> {
     }
 
     private getSubmissionInfo(): string {
-        if (this.props.labInfo.latest) {
-            return this.props.labInfo.latest.approved ? "Approved" : "Not approved";
+        if (this.props.assignment.latest) {
+            return this.props.assignment.latest.approved ? "Approved" : "Not approved";
```
Comment (meling) [inline public/src/pages/views/LabResultView.tsx]: Not Approved (capital A)

Comment (0xf8f8ff) [reply public/src/pages/views/LabResultView.tsx]: The method was not even used anymore, so I've just removed it.

File: web/autograder_service.go (hunk 1 above)
Comment (0xf8f8ff) [reply web/autograder_service.go]: Done.

**Pull request merged**